pkg/patterns/declarative/image.go

```diff
@@ -40,11 +43,14 @@ func applyImageRegistry(ctx context.Context, operatorObject DeclarativeObject, m
 			manifestItem.Kind == "StatefulSet" || manifestItem.Kind == "Job" ||
 			manifestItem.Kind == "CronJob" {
 			if registry != "" {
+				log.WithValues("manifest", manifestItem).WithValues("registry", registry).V(1).Info("applying image registory to manifest")
 				if err := manifestItem.MutateContainers(applyPrivateRegistryToContainer(registry)); err != nil {
 					return fmt.Errorf("error applying private registry: %v", err)
 				}
 			}
 			if secret != "" {
+				//TODO: Is it better not to show contents of secret for security reason?
```

Comment:
It's fine, this is a name of a secret, not a secret value. Safe to emit and drop this comment.

Reply:
Thank you for comment.
Done.